src/index.html

```diff
@@ -1,13 +1,376 @@
+
+          <div class="explore__content">
+            <div class="explore__image">
+              <div class="explore__image-container">
+                <img
+                  src="images/met/painting.jpg"
+                  alt="old paiting"
+                  class="explore__picture"
+                >
+              </div>
+            </div>
+            <h3 class="explore__description">Virtual group tours</h3>
+            <h4 class="explore__description-subtitle">
+              Request a guided tour, reserve admission
+              and lead your own small group, or request
+              a live virtual tour led by a Museum guide.
+            </h4>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section page__section-title-secondary" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">contact us</h2>
+          <div class="contacts">
+            <div class="grid__item--1-4">
+              <div class="contacts__info">
+                <h4 class="contacts__name">the met fifth avenue</h4>
+                <div class="contacts__title">phone</div>
+                <a href="tel:+1 212-535-7710" class="contacts__data">+1 212-535-7710</a>
+                <div class="contacts__title">address</div>
+                <div>
+                  <a
+                    href="https://www.google.com/maps/place/Metropolitan+Museum+of+Art/@40.7794365,-73.9677233,17z/data=!3m2!4b1!5s0x89c25896e10b5523:0x6c1168e355509b8b!4m5!3m4!1s0x89c25896f660c26f:0x3b2fa4f4b6c6a1fa!8m2!3d40.7794366!4d-73.963244"
+                    class="contacts__data"
+                    target="_blank"
+                  >
+                    1000 Fifth Avenue
+                    <br>
+                    New York, NY 10028
+                  </a>
+                </div>
+              </div>
+            </div>
+
+            <div class="grid__item--5-7">
+              <div class="contacts__info">
+                <h4 class="contacts__name ">The Met Cloisters</h4>
+                <div class="contacts__title">phone</div>
+                <a href="tel:+1 212-535-7710" class="contacts__data">+1 212-923-3700</a>
+                <div class="contacts__title">address</div>
+                <div>
+                  <a
+                    href="https://www.google.com/maps/place/The+Cloisters/@40.8648628,-73.9339214,17z/data=!3m1!4b1!4m5!3m4!1s0x89c2f40170f2a62b:0x9191e7fa95df29ad!8m2!3d40.8648628!4d-73.9317274"
+                    class="contacts__data"
+                    target="_blank"
+                  >
+                    99 Margaret Corbin Drive
+                    <br>
+                    Fort Tryon Park
+                    <br>
+                    New York, NY 10040
+                  </a>
+                </div>
+              </div>
+            </div>
+
+            <div class="grid__item--8-12">
+              <div class="contacts_info">
+                <form action="/">
```

Comment:
```suggestion
                <form action="#">
```
Try this. Also, would be great to use method post, to avoid passing the form data through URL
![image](https://user-images.githubusercontent.com/96409563/205503066-052bdba2-86d8-4757-aefc-cb2d08e05117.png)

Reply:
Hi Oleksander, thank you for your helop. About the scrolling in the menu, I think I need JS to fix that, but I have *no idea* how to do it, because I haven't studied that yet. Can you please help me?